docs/source/developing.rst

```diff
@@ -40,14 +41,17 @@ To develop ixdat, you will need to use git and github. This means
 
     pip install --upgrade ixdat
 
-- Check out the branch you want to work from. Note, this is also how to *use* a feature that is under development.::
+- Switch to the branch you want to work from. Note, this is also how to *use* a feature that is under development.::
+
+    git switch branch_to_work_from
 
-    git checkout branch_to_work_from
+  Usually it makes most sense to work from the main branch::
```

Comment:
Maybe the formulation should be "Usually ... to **start** work from ..."

Reply:
so as to not cause confusion below when you fire up the feature branch

Reply:
Usually it makes most sense to start work by branching from the main branch